src/components/table/table_pagination/table_pagination.tsx

```diff
@@ -114,8 +120,8 @@ export const EuiTablePagination: FunctionComponent<EuiTablePaginationProps> = (
     () =>
       itemsPerPageOptions.map((itemsPerPageOption) => (
         <EuiContextMenuItem
+          {...setActiveProps(itemsPerPageOption)}
```

Comment:
This doesn't need an entire object, just IMO. I'd just repeat the ternary
```suggestion
          icon={itemsPerPageOption === itemsPerPage ? 'check' : 'empty'}
          aria-current={itemsPerPageOption === itemsPerPage || undefined}
```